src/BuildScriptGenerator/DotNetCore/ProjectFileProviders/ProbeAndFindProjectFileProvider.cs

```diff
@@ -63,26 +64,85 @@ public string GetRelativePathToProjectFile(RepositoryContext context)
 
             var webAppProjects = new List<string>();
             var azureFunctionsProjects = new List<string>();
+            var azureBlazorWasmProjects = new List<string>();
             var allProjects = new List<string>();
+            bool functionProjectExists = false;
+            bool webAppProjectExists = false;
+            bool blazorWasmProjectExists = false;
+
             foreach (var file in projectFiles)
             {
                 allProjects.Add(file);
                 if (ProjectFileHelpers.IsAspNetCoreWebApplicationProject(sourceRepo, file))
                 {
-                    webAppProjects.Add(file);
+                    if (ProjectFileHelpers.IsAzureBlazorWebAssemblyProject(sourceRepo, file))
```

Comment:
hmm, this looks incorrect. Currently wasm project supports `netstandard2.0` and we look for `netcoreapp` in case of webapplication project. So if this code is nested under webapplication project, it would not be executed. Am i missing soemthing? can you can write unit tests for this scenario?

Reply:
yeah ... i wanted to discuss this .. i think we need to add more logic to current implementation of IsAspNetCoreWebApplicationProject which returns true if it IsOfSdkProjectType. Both blazor wasm and web apps have Microsoft.Net.Sdk.Web in their csproj

Reply:
Ok, a project can be a wasm one only if it has `Microsoft.Net.Sdk.Web` and `netstandard` prefix?

Reply:
adding to my previous comment: and `Microsoft.AspNetCore.Components.WebAssembly`